src/components/QualityEvaluation/QualityEvaluation.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { useField } from "formik";
+import { CSSProperties, useMemo } from "react";
+import { useTranslation } from "react-i18next";
+import styled from "@emotion/styled";
+import { colors, spacing } from "@ndla/core";
+import { IQualityEvaluation } from "@ndla/types-backend/draft-api";
+import { Text } from "@ndla/typography";
+import { QualityEvaluationFormValues, gradeItemStyles } from "./QualityEvaluationForm";
+import QualityEvaluationModal from "./QualityEvaluationModal";
+
+const qualityEvaluationOptions: { value: number; color: string }[] = [
+  { value: 1, color: colors.support.green },
+  { value: 2, color: "#C3D060" },
+  { value: 3, color: colors.support.yellow },
+  { value: 4, color: "#CF9065" },
+  { value: 5, color: colors.assessmentResource.dark },
+];
+
+const FlexWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  gap: ${spacing.xsmall};
+`;
+
+const GradeItem = styled.div`
+  ${gradeItemStyles}
+  cursor: default;
+`;
+
+const StyledNoEvaluation = styled(Text)`
+  color: ${colors.brand.greyMedium};
+  font-style: italic;
+`;
+
+interface Props {
+  qualityEvaluation: IQualityEvaluation | undefined;
```

Comment:
Vil dette kunne fungere med litt mindre kode, slik?
```suggestion
  qualityEvaluation?: IQualityEvaluation;
```

Reply:
Det vil ikke tilsvare det samme 😅 Hvis det endres så trenger man ikke sende ned propen i det hele tatt, sånn jeg har typet det så betyr det bare at qualityEvaluation kan være undefined

Reply:
Tenkte mest på innholdet / bruken i komponenten, gitt innholdet i propertyet. Så lenge det er et bevisst formål at prop alltid MÅ oppgis, bør det naturligvis stå slik det gjør, ja 🐈